File: tests/rolling_pairwise_beta_test.py
import numpy as np
from taflow import RollingPairwiseBeta
def test_rolling_pairwise_beta_lifecycle():
    s=RollingPairwiseBeta(np.array([],float),np.array([],float),2);s.extend([1,2],[2,4]);assert s.value is not None;s.reset();assert len(s)==0

File: crates/taflow-core/src/indicators/rolling_pairwise_beta_test.rs
use super::rolling_pairwise_beta::RollingPairwiseBeta;
#[test]
fn lifecycle() {
    let mut s = RollingPairwiseBeta::new(2).unwrap();
    s.append(1.0, 2.0);
    assert!(s.append(2.0, 4.0).is_some());
    s.reset();
    assert!(s.value().is_none());
}
